test/built-ins/Number/S9.3.1_A2_U180E.js

```diff
@@ -0,0 +1,33 @@
+// Copyright 2009 the Sputnik authors.  All rights reserved.
```

Comment:
needs to update the year and author. Other new files on this branch have the same issue.